api/src/main/java/ai/djl/training/DefaultTrainingConfig.java

```diff
@@ -39,15 +39,12 @@
 
     /**
      * Creates an instance of {@code DefaultTrainingConfig} with the given {@link Loss}. {@code
-     * DefaultTrainingConfig} creates a default {@link TrainingConfig} with the {@link
-     * XavierInitializer} as initialiser, {@link Adam} as optimiser, and the given {@link Loss}. The
-     * evaluators and listeners are left to the user's discretion.
+     * DefaultTrainingConfig} creates a default {@link TrainingConfig}, {@link Adam} as optimiser,
+     * and the given {@link Loss}. The evaluators and listeners are left to the user's discretion.
      *
      * @param loss the loss to use for training
      */
     public DefaultTrainingConfig(Loss loss) {
-        // Defaults to initializer defined in https://arxiv.org/abs/1502.01852
-        this.initializer = new XavierInitializer(RandomType.GAUSSIAN, FactorType.IN, 2);
```

Comment:
Can you keep this as the default initializer?

Reply:
XavierInitializer is already a default initializer. You can check Parameter Type weight

Reply:
I copied the parameter of XavierInitializer from TensorFlow. Let me know if it is really optimal

Reply:
I am not sure the best way to present this. Right now, there are two papers on Xavier initializer (that I know of at least). The default parameters in `new XavierInitializer()` are the first paper because it also named the initializer. The `new XavierInitializer(RandomType.GAUSSIAN, FactorType.IN, 2)` are from the second which improved upon it. We probably should make that a constant in XavierInitializer or something, but I think we should keep our default initializer to the second paper rather than reverting it to the first paper.

Reply:
done!